src/components/TheHeader.vue

```diff
@@ -42,13 +53,22 @@ export default {
     donateUrl: {
       type: String,
       default: null
+    },
+    title: {
+      type: String,
+      default: null
     }
   }
 }
 </script>
 
 <style lang="scss">
 $z-index-header: 5000;
+
+.sbdocs-xsmall-container {
```

Comment:
why is this in here? this is part of the storybook specific styles so you can take this out

Reply:
Probably a misunderstanding on my end on how this style works -- when I remove it, it reverts back to the default width. Changing the div name in the template has no effect on it.